src/main/scala/com.snowplowanalytics.sauna/actors/Mediator.scala

```diff
@@ -145,6 +146,7 @@ class Mediator(saunaSettings: SaunaSettings) extends Actor {
               original match {
                 case l: LocalFilePublished => original.observer ! Observer.DeleteLocalFile(l.file)
                 case s: S3FilePublished => original.observer ! Observer.DeleteS3Object(s.path, s.s3Source)
+                case r: KinesisRecordReceived =>
```

Comment:
I'd write it as `case r: KinesisRecordReceived =>()` (Unit value in the end) to explicitly conform this partial function type. This is minor though.